ast/commandflag/flags.go

```diff
@@ -4,6 +4,10 @@ import (
 	"time"
 )
 
+// NOTE: Any new flags must be accompanied by a `VERSION --feature-flag ...` change.
```

Comment:
> Any new flags must be accompanied by a `VERSION --feature-flag ...` change.

What sort of change? Do you mean a changelog entry?

Reply:
does 

```
// NOTE: Any new command flags must be accompanied by the introduction of a new `VERSION` feature flag.`
```

 read better?

This is not related to CHANGELOG entries (but yes those are important too, but seems out of scope of this PR since a CHANGELOG entry should be added for all changes/fixes/etc, we could improve docs for CHANGELOG requirements if you think that's important)